### PR TITLE
fix(elements/ino-input): prevent label cut-off in outline variant

### LINKED ISSUE #1164: ino-input: Outline variant truncates label after a fixed width
## Description
An issue has been identified in the outlined variant of our `ino-input` component, where labels exceeding a certain length are not being handled correctly. This problem appears to be a bug introduced during a recent design update. Please refer to the provided screenshot for a visual representation of the issue.

## Steps to Reproduce

1. [Open the storybook page](https://elements.inovex.de/version/latest/?path=/docs/input-ino-input--playground) of our `<ino-input>`
2. Set outline to true
3. Click into the input field
4. Observe that the label is not displayed or handled appropriately based on the length.

## Expected Behavior
The outlined variant of the `ino-input` component should gracefully handle labels of varying lengths, ensuring proper visibility and display without any unintended issues.

## Actual Behavior
Labels that surpass a certain length in the outlined variant of `ino-input` are not handled correctly, leading to undesired visual anomalies.

## Additional Information
Image

## Acceptance Criteria

- Identify and rectify the issue causing label length limitations in the outlined variant of `ino-input`.
- Ensure that the updated handling accommodates labels of varying lengths without visual anomalies.
- Conduct thorough testing to confirm that the adjustment does not introduce regressions or impact the overall functionality of the component.

### PR DESCRIPTION
**Pull request by MariaLStefan**
Closes #1164 

## Proposed Changes

- fixes truncating label on `ino-input`

## showcase

https://github.com/inovex/elements/assets/103122411/4bb9aba1-56fc-4ff1-9f9b-4676a0a8fdd5

### CODE REVIEW

File: packages/elements/src/components/ino-input/ino-input.scss [hunk 1]
```diff
@@ -199,6 +199,10 @@ ino-input .mdc-text-field--outlined {
     @include textfield.focused-outline-color(theme.$error);
   }
 
+  & .mdc-floating-label:not(.mdc-floating-label--float-above) {
```
Comment (janivo) [inline packages/elements/src/components/ino-input/ino-input.scss:202]: Does not seem to be fixed quite yet:

image

These changes seem to have no effect when the label is floating above.

**Review by janivo — CHANGES_REQUESTED** [review]

**Review by janivo — CHANGES_REQUESTED** [review]
Seems like it still not quite right. In the storybook:

image

**Comment by BenPag** [comment]
> Seems like it still not quite right. In the storybook:
> 
> image

To late yesterday... I fogot to adjust some parameters for optimal spacing...

**Comment by janivo** [comment]
@BenPag still the same 🤔 
image

**Review by janivo — APPROVED** [review]

**Pull request merged**